Maquetado/scss/bookDetail.scss

```diff
@@ -2,88 +2,117 @@
+  align-items: center;
+  background-color: $seashell;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding-top: 25px;
+  padding-bottom: 28%;
+
+
+  .backButtonContainer {
     display: flex;
-    flex-direction: column;
-    gap: 15px;
-    padding-top: 25px;
-    padding-bottom: 28%;
+    flex-direction: row-reverse;
+    position: relative;
+    right: 38%;
 
+    .backButton {
+      font-size: 18px;
+    }
 
-    .backButtonContainer {
-      display: flex;
-      flex-direction: row-reverse;
-      margin-right: 80%;
+    .backArrowImage {
+      margin-right: 10px;
+    }
+  }
 
-      .backButton {
-        font-size: 18px;
-      }
+  .bookContainer {
+    display: flex;
+    background-color: $white;
+    box-shadow: 4px 4px 9px -4px rgba($black, 1);
+    margin-top: 20px;
+    padding: 30px;
+    width: 65%;
 
-      .backArrowImage {
-        margin-right: 10px;
-      }
+    .bookImage {
+      height: 385px;
+      position: relative;
+      width: 250px;
+      margin-right: 30px;
     }
 
-    .bookContainer {
-      display: flex;
-      background-color: $white;
-      box-shadow: 4px 4px 9px -4px rgba($black, 1);
-      width: 65%;
+    &::after {
+      content: '';
+      background: url('./resources/img/badge.png');
+      background-size: cover;
+      height: 80px;
+      left: 33.5%;
+      position: absolute;
+      transform: rotate(20deg);
+      width: 55px;
+    }
 
-      .bookImage {
-        height: 385px;
-        padding: 20px;
-        position: relative;
-        width: 250px;
-      }
+    .mainTitle {
+      @include bookDetailfontStyle();
+      border-bottom: 4px solid $conifer;
+      font-size: 32px;
+      line-height: 38px;
+      margin-bottom: 32px;
+      padding-bottom: 10px;
+    }
 
-      &::after {
-        content: '';
-        background: url('./resources/img/badge.png');
-        background-size: cover;
-        height: 80px;
-        left: 30%;
-        margin-top: 6px;
-        position: absolute;
-        transform: rotate(20deg);
-        width: 55px;
-      }
+    .title {
+      @include bookDetailfontStyle();
+      font-size: 20px;
+      margin-bottom: 30px;
+      line-height: 24px;
+    }
 
-      .textContainer {
-        padding: 20px;
-        width: 80%;
-      }
+    .bookGenre {
+      color: $mediumGray;
+      font-family: Roboto;
+      font-size: 24px;
+      font-weight: 700;
+      line-height: 28px;
+    }
 
-      .mainTitle {
-        @include bookDetailfontStyle();
-        border-bottom: 4px solid $conifer;
-        font-size: 32px;
-        line-height: 38px;
-        margin-bottom: 35px;
-      }
+    .bookDetail {
+      color: $mediumGray;
+      font-family: Roboto;
+      font-size: 20px;
+      line-height: 24px;
+    }
+  }
+}
 
-      .title {
-        @include bookDetailfontStyle();
-        font-size: 20px;
-        margin-bottom: 30px;
-        line-height: 24px;
-      }
+@media only screen and (max-width: 1024px) {
```

Comment:
You can define the breakpoint value in a variable, so in case the the breakpoint changes you only have to change it once